modules/logging-es-storage-considerations.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-deploying.adoc
+
+[id="logging-es-storage-considerations_{context}"]
+= Storage considerations for Elasticsearch
+
+A persistent volume is required for each Elasticsearch deployment configuration. On {product-title} this is achieved using persistent volume claims.
+
+[NOTE]
+====
+If you use a local volume for persistent storage, do not use a raw block volume, which is described with `volumeMode: block` in the `LocalVolume` object. Elasticsearch cannot use raw block volumes.
+====
+
+The OpenShift Elasticsearch Operator names the PVCs using the Elasticsearch resource name.
+
+Fluentd ships any logs from *systemd journal* and **/var/log/containers/*.log** to Elasticsearch.
```

Comment:
Are these UI elements (which would be bold, as you have here) or object references (backticks)?

Reply:
Not sure - this was pre-existing content that was moved from elsewhere in the doc, so I will leave it as is for now.